mad_gui/components/dialogs/plugin_selection/load_data_dialog.py

```diff
@@ -301,8 +294,163 @@ def _handle_video_file(self, return_dict, loader):
             return_dict["sync_file"] = sync_file
         return return_dict
 
-    def get_data(self) -> Optional[Tuple[Dict[str, Dict[str, Any]], BaseImporter]]:
+    def get_data(self) -> Union[Tuple[Dict[str, Dict[str, Any]], BaseFileImporter], Tuple[None, None]]:
         """Run this dialog and return the data, that was selected by the user."""
         if self.exec_():
             return self.final_data_, self.loader_
         return None, None
+
+
+class FromPluginLoaderDialog(QDialog):
```

Comment:
```suggestion
class FromDatasetLoaderDialog(QDialog):
```
or even remove the `From`. The reason why I would suggest this is, that also the FileLoaderDialog actually also makes use of plugins and therefore, I think `FromPluginLoaderDialog` is not pinpointed